### DISCUSSION TITLE
Welcome to VueStart Discussions!

### DISCUSSION THREAD

**Discussion by agabor** [BootGen/VueStart#55] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Send feature requests
  * Share feedback. We welcome both positive and negative feedback (especially negative!). We learn more from criticism than we do from compliments 💡.

**Comment by ghost** [comment]
Hello! Here!

**Reply by agabor** [reply]
Hello
